KenseiWars/Assets/Scripts/Combat/Components/BasicAbilities/BasicAttackComponent.cs

```diff
@@ -40,6 +40,8 @@ void OnTriggerEnter2D(Collider2D other)
                 {
                     enemy.IsHit();
                 }
+
+                break;
```

Comment:
Am pus break pentru optimizare. In codul din acel fisier iteram printr-o list sa vad daca obiectul cu care am facut coliziune are tag, daca il gaseam nu ma opream, asa ca acum cand il gasesc ies din alte cautari.

Reply:
👍